src/sagemaker/jumpstart/model.py

```diff
@@ -36,109 +28,54 @@ class JumpStartModel(Model):
 
     def __init__(
         self,
-        ############## ▼ args unique to JumpStartModel ▼ ##############
         model_id: str,
-        model_version: Optional[str] = "*",
+        model_version: Optional[str] = None,
         instance_type: Optional[str] = None,
-        region: Optional[str] = JUMPSTART_DEFAULT_REGION_NAME,
-        ############## ▲ args unique to JumpStartModel ▲ ##############
-        ############## ▼ args passed to base Model class ▼ ##############
+        region: Optional[str] = None,
         image_uri: Optional[str] = None,
-        model_uri: Optional[str] = None,
-        script_uri: Optional[str] = None,
+        model_data: Optional[str] = None,
+        source_dir: Optional[str] = None,
         entry_point: Optional[str] = None,
         env: Optional[Dict[str, str]] = None,
-        predictor_cls: Predictor = "JumpStartPredictor",
+        predictor_cls: Optional[Predictor] = None,
         **kwargs,
```

Comment:
except for `model_id` and `model_version`, all other arguments should be present in the parent [constructor](https://github.com/aws/sagemaker-python-sdk/blob/409546d47eaa642f9598d0bbf966ea302e87f638/src/sagemaker/model.py#L119). Please remove all arguments that are not present (eg `instance_type`)

Please also expose at a minimum the parameters we use:
- `role`
- `sagemaker_session`
- `predictor_cls`
- `vpc_config`
- `enable_network_isolation`
- `model_kms_key`
- `code_location`

question: should `model_id` be optional so that we can support just passing a `task` in the future?